bgpd/bgp_network.c

```diff
@@ -360,10 +360,13 @@ static int bgp_accept(struct thread *thread)
 
 	sockunion_init(&su);
 
+	bgp = bgp_lookup_by_name(listener->name);
```

Comment:
`struct bgp_listener` has a ->bgp pointer.  Why do we need to look it up?

Reply:
```
        /* this socket needs a change of ns. record bgp back pointer */    
        if (bgp->vrf_id != VRF_DEFAULT && vrf_is_backend_netns())    
                listener->bgp = bgp;   
```